src/legacy/status_im/ui/screens/browser/empty_tab/views.cljs

```diff
@@ -144,7 +143,7 @@
          :container-style     styles/input-container-style
          :accessibility-label :dapp-url-input
          :return-key-type     :go}]
-       [components/separator-dark]
+       [react/view {:style {:margin-top 8}}]
```

Comment:
I removed the separator component and just added in a quick gap component, not sure how it looks but does any one know if we really care? 
When is browser scoped for the redesign? 
@cammellos @mohsen-ghafouri @Parveshdhull any ideas? 🤔

Reply:
> does any one know if we really care?

At this stage probably not

> not sure how it looks

It should be ok, but if you want to be sure, just use the same styles as the separator-dark component itself.

`{:height 1 :background-color (colors/alpha colors/black 0.1)}`

Reply:
true true, thanks @Parveshdhull ! 🙏